gk/main.c

```diff
@@ -478,6 +479,75 @@ get_block_idx(struct gk_config *gk_conf, unsigned int lcore_id)
 	return 0;
 }
 
+static bool
+is_flow_expired(struct flow_entry *fe, uint64_t request_timeout_cycles)
```

Comment:
The callers of `is_flow_expired()` call it in loops, so it's better to receive `now` as a parameter to have a single time reference throughout the loop.